<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NeedsDock</class>
 <widget class="QDockWidget" name="NeedsDock">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Needs</string>
  </property>
  <widget class="QWidget" name="contents">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTreeView" name="focus_view">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContents</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="SearchFilterTreeView" name="need_view" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="match_all_checkbox">
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;If checked filter dwarves that match &lt;span style=&quot; font-style:italic;&quot;&gt;all&lt;/span&gt; the selected needs, otherwise filter dwarves that match &lt;span style=&quot; font-style:italic;&quot;&gt;any&lt;/span&gt; of them.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string>Display only dwarves with all the selected needs</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SearchFilterTreeView</class>
   <extends>QWidget</extends>
   <header>searchfiltertreeview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
